<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Adjustment</class>
 <widget class="QDialog" name="Adjustment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="site_LineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="adjustement_LineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>231</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_PushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>560</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="release_PushButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>560</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Release</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="site_Label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Site : </string>
   </property>
  </widget>
  <widget class="QLabel" name="date_Label">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>20</y>
     <width>59</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Date : </string>
   </property>
  </widget>
  <widget class="QLabel" name="adjustement_Label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Adjustement : </string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>721</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="siteName_LineEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>60</y>
     <width>291</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="date_dateEdit">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>20</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="stock_Label">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>160</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Stock :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="stock_LineEdit">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>160</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="quantity_Label">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>130</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Quantit√© : </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="quantity_LineEdit">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>130</y>
     <width>241</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="ItemWidget" name="itemWidget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>411</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="ItemTable" name="tableWidget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>721</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ItemTable</class>
   <extends>QWidget</extends>
   <header>PromotedClass/itemtable.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ItemWidget</class>
   <extends>QWidget</extends>
   <header>PromotedClass/itemwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>site_LineEdit</tabstop>
  <tabstop>cancel_PushButton</tabstop>
  <tabstop>release_PushButton</tabstop>
  <tabstop>siteName_LineEdit</tabstop>
  <tabstop>date_dateEdit</tabstop>
  <tabstop>adjustement_LineEdit</tabstop>
  <tabstop>stock_LineEdit</tabstop>
  <tabstop>quantity_LineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
